tests/foreman/ui_airgun/test_sync.py

```diff
@@ -0,0 +1,177 @@
+"""Test class for Custom Sync UI
+
+:Requirement: Sync
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo import manifests
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.constants import (
+    DISTRO_RHEL6, DISTRO_RHEL7,
+    FAKE_1_YUM_REPO,
+    FEDORA23_OSTREE_REPO,
+    REPOS,
+    REPOSET,
+    PRDS,
+)
+from robottelo.decorators import (
+    fixture,
+    run_in_one_thread,
+    skip_if_not_set,
+    tier2,
+    upgrade,
+)
+from robottelo.decorators.host import skip_if_os
+from robottelo.products import (
+    RepositoryCollection,
+    RHELCloudFormsTools,
+    SatelliteCapsuleRepository,
+)
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def module_custom_product(module_org):
+    return entities.Product(organization=module_org).create()
+
+
+@fixture(scope='module')
+def org_manifest(module_org):
+    manifests.upload_manifest_locked(module_org.id)
```

Comment:
redundant as such fixture won't be used anywhere. Pls add this step to `module_org` instead.

Reply:
@abalakh I am not sure about this as some tests are running in one thread and use manifest, others run in parallel and do not use manifest.

Reply:
But you're updating the same organization, so module_org will receive manifest anyways.

Reply:
no , the other tests are running not in one thread and will not need manifest